app/models/course.rb

```diff
@@ -18,7 +18,7 @@ class Course < ActiveRecord::Base
 
   belongs_to :instance, inverse_of: :courses
   has_many :enrol_requests, inverse_of: :course, dependent: :destroy
-  has_many :course_users, inverse_of: :course, dependent: :destroy
+  has_many :course_users, -> { order(:name) }, inverse_of: :course, dependent: :destroy
```

Comment:
Better to order this at the groups page. This will add a default scope to `course_user`, which will because problematic if we merge with other scopes.